presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1500,6 +1501,19 @@ public double getDefaultJoinSelectivityCoefficient()
         return defaultJoinSelectivityCoefficient;
     }
 
+    @Config("optimizer.default-cte-producer-replication-coefficient")
+    @ConfigDescription("The replication coefficient for CTE producers which aids the cost calculator in determining the multiplier that quantifies the expense of writing operations")
```

Comment:
nit: perhaps a simpler doc string "Replication coefficient for costing write operations during CTE materialization"